src/main/java/uk/gov/hmcts/ccd/domain/service/callbacks/CallbackService.java

```diff
@@ -22,79 +23,118 @@
                            @Qualifier("restTemplate") final RestTemplate restTemplate) {
         this.securityUtils = securityUtils;
+        this.defaultCallbackRetryIntervalsInSeconds = applicationParams.getCallbackRetryIntervalsInSeconds();
+        this.defaultCallbackTimeoutInMillis = applicationParams.getCallbackReadTimeoutInMillis();
         this.restTemplate = restTemplate;
     }
 
-    // The retry will be on seconds T=1 and T=3 if the initial call fails at T=0
-    @Retryable(value = {CallbackException.class}, maxAttempts = 3, backoff = @Backoff(delay = 1000, multiplier = 3))
     public Optional<CallbackResponse> send(final String url,
+                                           final List<Integer> callbackRetryTimeouts,
                                            final CaseEvent caseEvent,
-                                           final CaseDetails caseDetailsBefore,
-                                           final CaseDetails caseDetails,
-                                           final Boolean ignoreWarning) {
+                                           final CaseDetails caseDetails) {
+        return send(url, callbackRetryTimeouts, caseEvent, null, caseDetails);
+    }
 
-        if (url == null || url.isEmpty()) {
-            return Optional.empty();
-        }
-        final CallbackRequest callbackRequest = new CallbackRequest(caseDetails, caseDetailsBefore, caseEvent.getId(), ignoreWarning);
-        final Optional<ResponseEntity<CallbackResponse>> responseEntity = sendRequest(url, CallbackResponse.class, callbackRequest);
-        return responseEntity.map(re -> Optional.of(re.getBody())).orElseThrow(() -> {
-            LOG.warn("Unsuccessful callback to {} for caseType {} and event {}", url, caseDetails.getCaseTypeId(), caseEvent.getId());
-            return new CallbackException("Callback to service has been unsuccessful for event " + caseEvent.getName());
-        });
+    public Optional<CallbackResponse> send(final String url,
+                                           final List<Integer> callbackRetryTimeouts,
+                                           final CaseEvent caseEvent,
+                                           final CaseDetails caseDetailsBefore,
+                                           final CaseDetails caseDetails) {
+        ResponseEntity<CallbackResponse> responseResponseEntity = send(url, callbackRetryTimeouts, caseEvent,
+            caseDetailsBefore, caseDetails, CallbackResponse.class, false);
+        return Optional.ofNullable(responseResponseEntity.getBody());
     }
 
-    // The retry will be on seconds T=1 and T=3 if the initial call fails at T=0
-    @Retryable(value = {CallbackException.class}, maxAttempts = 3, backoff = @Backoff(delay = 1000, multiplier = 3))
+    @SuppressWarnings("javasecurity:S5145")
     public <T> ResponseEntity<T> send(final String url,
+                                      final List<Integer> callbackRetryTimeouts,
```

Comment:
`callbackRetryTimeouts` is not consistent with the name chosen for the default setting, which is `defaultCallbackRetryIntervalsInSeconds`. A bit confusing

Reply:
As mentioned above this is actually desired.
`callbackRetryTimeouts` is the callbacks duration dynamically configurable by spreadsheet. `defaultCallbackRetryIntervalsInSeconds` on the other hand is the default interval of time in seconds between callbacks (it is not callback duration) that is static here and comes from the application properties.

Reply:
yep got it now got confused by the `ccd.callback.retries` left in the code